src/js/components/about/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import Link from '../router/link';
+
+function About() {
+    return (
+        <div className="about-content">
+            <h2>About</h2>
+            <div>
+                <h3>What is this?</h3>
+                <p>
+                    Personal project as a way to learn while coding. The main focus was learning
+                    React, Redux, Test Driven Development with Jest and as a final step NodeJs.
+                </p>
+                <h3>What does the app do?</h3>
+                <p>
+                    The app fetches data from DogAPI and shows it in a simple way as an images list.
+                    The list can be filtered, contains a lazy load component for the dogs images and
+                    you can submit a dog picture through a form. It implements a handmade router and
+                    manages state with Redux and Context.
```

Comment:
This implies that both your app and router use redux and context. Your app uses Redux, while your Router uses context, the difference is importante because you can take your router into any other React app and it should work, while your app is coupled to Redux.

Reply:
Done.